Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -9246,6 +9246,111 @@ def get_template_stacks(
     return PanoramaCommand.get_template_stacks(topology, device_filter_string)
 
 
+def get_global_counters(topology: Topology, device_filter_string: str = None) -> ShowCounterGlobalCommmandResult:
+    """
+    Gets global counter information from all the PAN-OS firewalls in the topology
+    :param topology: `Topology` instance !no-auto-argument
+    :param device_filter_string: String to filter to only show specific hostnames or serial numbers.
+    """
+    result: ShowCounterGlobalCommmandResult = FirewallCommand.get_counter_global(topology, device_filter_string)
+    return result
```

Comment:
```suggestion
    return FirewallCommand.get_counter_global(topology, device_filter_string)
```

Please do it to all of the functions below as well